acm.tf

```diff
@@ -26,3 +26,39 @@ resource "aws_iam_policy" "pdm_dataset_acm" {
   description = "Allow export of PDM Dataset Generator certificate"
   policy      = data.aws_iam_policy_document.pdm_dataset_acm.json
 }
+
+data "aws_iam_policy_document" "pdm_certificates" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${local.mgt_certificate_bucket}*",
+      "arn:aws:s3:::${local.env_certificate_bucket}/*",
+    ]
+  }
```

Comment:
you can remove this statement, it is covered by the statement below

Reply:
removed